File: src/net/merayen/elastic/ui/objects/top/views/transportview/TransportView.java
package net.merayen.elastic.ui.objects.top.views.transportview;

import net.merayen.elastic.ui.objects.top.views.View;

public class TransportView extends View {
	private TransportViewBar bar = new TransportViewBar();
	private TransportViewButtons buttons = new TransportViewButtons();

	@Override
	public void onInit() {
		super.onInit();
		add(bar);
		this.add(buttons);
		buttons.getTranslation().y = 20;
	}

	@Override
	public View cloneView() {
		return new TransportView();
	}

}

File: src/net/merayen/elastic/ui/objects/top/views/transportview/TransportView.kt
package net.merayen.elastic.ui.objects.top.views.transportview

import net.merayen.elastic.ui.objects.top.views.View

class TransportView : View() {
	private val bar = TransportViewBar()
	private val buttons = TransportViewButtons()

	override fun onInit() {
		super.onInit()
		add(bar)
		this.add(buttons)
		buttons.translation.y = 20f
	}

	override fun cloneView(): View {
		return TransportView()
	}

}
